src/diffusers/pipelines/i2vgen_xl/pipeline_i2vgen_xl.py

```diff
@@ -98,12 +99,13 @@ class I2VGenXLPipelineOutput(BaseOutput):
     Output class for image-to-video pipeline.
 
     Args:
-        frames (`List[np.ndarray]` or `torch.FloatTensor`)
-            List of denoised frames (essentially images) as NumPy arrays of shape `(height, width, num_channels)` or as
-            a `torch` tensor. The length of the list denotes the video length (the number of frames).
+        frames (`torch.Tensor`, `np.ndarray`, or List[List[PIL.Image.Image]]):
+        Nested list of length `batch_size` with denoised PIL image sequences of length `num_frames`,
+        NumPy array of shape `(batch_size, num_frames, channels, height, width,
+        Torch tensor of shape `(batch_size, num_frames, channels, height, width)`.
```

Comment:
```suggestion
            List of video outputs - It can be a nested list of length `batch_size,` with each sub-list containing denoised 
    PIL image sequences of length `num_frames.` It can also be a NumPy array or Torch tensor of shape 
   `(batch_size, num_frames, channels, height, width)`
```